blog/themes/3mdeb/layouts/authors/list.html

```diff
@@ -39,16 +39,26 @@ <h2>Social media <strong>profiles</strong></h2>
                         <a href="{{ . }}"><img class="social-icon" src="/img/linkedin.png" height="32" width="32"> Find {{ $.Params.name }} on LinkedIn</a>
                       </div>
                     {{ end }}
+                    {{ with .Params.reddit }}
+                    <p>
+                      <a href="{{ . }}"><img class="social-icon" src="/img/reddit.png" height="32" width="32"> Follow {{ $.Params.name }} on Reddit</a>
+                    </p>
+                    {{ end }}
+                    {{ with .Params.mastodon }}
+                    <p>
+                      <a href="{{ . }}"><img class="social-icon" src="/img/mastodon.png" height="32" width="32"> Find {{ $.Params.name }} on Mastodon </a>
+                    </p>
+                    {{ end }}
```

Comment:
I think there is an issue with icon alignment or text icon location.

![icons_alignment](https://github.com/3mdeb/news-and-ideas/assets/1071894/f9c9fb9d-6a6e-47ea-82e6-986b932fdd78)

Reply:
@pietrushnic fix div and rescalled images to 64x64 as the rest of icons: https://github.com/3mdeb/news-and-ideas/pull/575/commits/c50d8fea05871e9cd45b76a42772a4c9114ed38e